frontend/src/views/Apps/AppPassphrase/AppPassphrase.js

```diff
@@ -96,12 +96,15 @@ class AppPassphrase extends Component {
       const {privateApplicationData} = data;
       const {address, privateKey} = privateApplicationData;
 
+      const appData = PocketApplicationService.getApplicationInfo().data;
+
       PocketApplicationService.removeAppInfoFromCache();
       PocketApplicationService.saveAppInfoInCache({
         applicationID: applicationInfo.id,
         passphrase: passPhrase,
         address,
-        privateKey
+        privateKey,
+        data: appData
```

Comment:
Here, you are duplicating the saved data, at this moment the only data that we don't save is the chains, and this field(chains) is not required at this moment.

Reply:
Yeah, I removed the duplication, such information was added on the previous view